PROTOCOL.md

```diff
@@ -0,0 +1,99 @@
+# GraphQL over WebSocket Protocol
```

Comment:
Missing intro section to convert terms between packet names and actual strings on the network.